x-pack/plugins/observability/public/components/shared/exploratory_view/obsv_exploratory_view.tsx

```diff
@@ -89,6 +91,7 @@ export const reportTypesList: Array<{
   { reportType: 'core-web-vitals', label: CORE_WEB_VITALS_LABEL },
   { reportType: 'device-data-distribution', label: DEVICE_DISTRIBUTION_LABEL },
   { reportType: 'single-metric', label: SINGLE_METRIC_LABEL },
+  { reportType: 'heat-map', label: HEAT_MAP_LABEL },
```

Comment:
"Heatmap" as a closed compound word is better i.e. use `reportType: 'heatmap'` and so the relating variable names if any.